guides/doc-Configuring_virt_who_VM_Subscriptions/master.adoc

```diff
@@ -6,6 +6,31 @@ include::common/header.adoc[]
 
 = {ConfiguringVMSubscriptionsDocTitle}
 
+////
+DOC INFO POC
+
+Category: Managing hosts
+
+Published: Satellite, Orcharhino
+
+Persona: System administrator (probably corresponds to 'Admin' in Contributing.md)
```

Comment:
Can we avoid the word `probably` here? That might just increase the uncertainty factor for contributors trying to follow this guidance.

Why `probably`? Is there concern that your `System administrator` persona doesn't actually match the `Admin` persona in Contributing.md? That would be a big issue because we'd have two different personas, both called "administrator".

If the `Admin` persona's definition in Contributing.md doesn't match your intention for System administrator, please think about what's missing in it and include a commit to update it.

Reply:
As I wrote in the 'description' above, this POC needs refinement.